v2/processor.go

```diff
@@ -118,6 +120,43 @@ func NewPanicHandler(handler HandlerFunc) HandlerFunc {
 	}
 }
 
+// NewTracingHandler extracts the context from the message Application property if available, or from the existing context if not, and starts a span
+func NewTracingHandler(handler HandlerFunc) HandlerFunc {
+	return func(ctx context.Context, settler MessageSettler, message *azservicebus.ReceivedMessage) {
+		var err error
+		if message != nil && message.ApplicationProperties != nil {
+			ctx, err = ExtractMessageContext(message)
+			if err != nil {
+				// do something
+			}
+		}
+
+		ctx, span := tab.StartSpan(ctx, "")
```

Comment:
need to define a naming prefix for go-shuttle spans and keep it consistent.

Reply:
Ok made the change - I modeled the naming prefix off what I saw in the original go-shuttle, what do you think?